docs/src/pages/lab/slider/CustomizedSlider.js

```diff
@@ -12,12 +12,9 @@ const useStyles = makeStyles({
 });
 
 const StyledSlider = withStyles({
-  container: {
-    overflow: 'visible',
-  },
   thumb: {
-    height: 32,
-    width: 32,
+    height: 24,
+    width: 24,
```

Comment:
Slightly less ugly, and the focus indicator doesn't overflow the Paper. (Why Paper to start with?)

Reply:
I have created this demo using this baseline.

![Capture d’écran 2019-04-30 à 22 45 03](https://user-images.githubusercontent.com/3165635/56992117-9e35b900-6b99-11e9-9b94-afa95b9e218b.png)

Reply:
Paper makes a bit more sense with the labels present, but since they aren't...  And I still think that the thumb is disproportionately large when viewed outside of that context. Funnily enough, I had tried to give the bar rounded ends! (What is Pretto?)

Reply:
Pretto is a money loan service for real estate in France.